src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {'My name is '}
+        {name}
+      </h2>
+      { age && (
+      <p className="Person__age">
+        {'I am '}
+        {age}
+      </p>
+      )}
+
+      {isMarried ? (
+        <p className="Person__partner">
+          {partnerName}
```

Comment:
you can use literal, but this solution requires you to assign tenary function to variable